server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/Keywords.scala

```diff
@@ -135,7 +122,7 @@ class KeywordsFactory(val validator: KeywordsValidator, val filter: Keyword => B
 
     def fromCollection(keywords: Set[String]): Keywords = fromSet(keywords.flatMap(toKeyword.toKeyword))
 
-    private[model] def fromMap(mapKeywords: Map[String, Boolean]): Keywords = {
+    def fromMap(mapKeywords: Map[String, Boolean]): Keywords = {
```

Comment:
IMO you should return a Try instead of throwing

Reply:
> IMO you should return a Try instead of throwing

Not done?